### PR TITLE
Editing Applications

### PR DESCRIPTION
**Pull request by larkmullins**

### CODE REVIEW

File: ui/src/app/resources/applications/application.service.ts [hunk 1]
```diff
@@ -37,4 +38,19 @@ export class ApplicationService {
...
       .catch((error: any) => Observable.throw(error.json().error || 'Server error'));
   }
+  
+  editApplication(name: string, uuid: string): Observable<Application> {
+    const headers = new Headers({
+      'Content-Type': 'application/json',
+      'Authorization': 'Bearer ' + this.auth.getToken()
+    });
+    console.log(uuid);
+    console.log(name);
```
Comment (jlaswell) [inline ui/src/app/resources/applications/application.service.ts]: some extra `console.log`s

File: ui/src/app/resources/applications/application-details.component.ts [hunk 2]
```diff
@@ -34,6 +35,10 @@ export class ApplicationComponent implements OnInit {
         },
         err => console.error('There was an error: ' + err)
       );
+    this.editApplicationForm = new FormGroup({
+      name: new FormControl('', Validators.required),
+      uuid: new FormControl('', Validators.required)
```
Comment (jlaswell) [inline ui/src/app/resources/applications/application-details.component.ts]: Do we need the `uuid` or can we just get that from the current `application` instance?

**Comment by larkmullins** [comment]
@jlaswell this is ready for review

**Review by jlaswell — APPROVED** [review]

**Comment by jlaswell** [comment]
lgtm

**Pull request merged**